doc/ova/1.0.4/Preboarding_and_Onboarding.rst

```diff
@@ -0,0 +1,131 @@
+==========================
```

Comment:
This whole file seems to be a duplicate of [https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst.](https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst.)  We should not duplicate our documents.  Is there some reason to duplicate this instead of just making edits to that other copy?

Reply:
This is to have two separate Preboarding_and_Onboarding.rst to describe two different procedures. 
One is for OVA 2.0.0, at https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst, that uses https://<management IP>...
The other (added by this PR) is in doc/ova/1.0.4/, specifically for OVA 1.0.4 and older, that uses https://<management IP>:**28100**...
@mukul-seagate11 has a change request to update for 1.0.4 release. So I use this way to separate the 1.0.4 from 2.0.0.

Reply:
@wb-droid, as of now, we are keeping same wiki page for both 1.0.4 & 2.0.0 to avoid any confusion in future